bn_error.c

```diff
@@ -27,10 +27,10 @@ static const struct {
 /* return a char * string for a given code */
 const char *mp_error_to_string(int code)
 {
-   int x;
+   size_t x;
 
    /* scan the lookup table for the given message */
-   for (x = 0; x < (int)(sizeof(msgs) / sizeof(msgs[0])); x++) {
+   for (x = 0; x < (sizeof(msgs) / sizeof(msgs[0])); x++) {
```

Comment:
I like the changes by "refactor cast" (currently 00209f06ec69c0ed18f70919a87f6c15b3353f2c) a lot